projects/canopy/src/lib/brand-icon/brand-icons.stories.ts

```diff
@@ -12,6 +12,8 @@ import * as brandIconSet from './brand-icons.interface';
 export const brandIconsArray: Array<brandIconSet.BrandIcon> = [
   brandIconSet.lgBrandIconSun,
   brandIconSet.lgBrandIconPiggyBank,
+  brandIconSet.lgBrandIconChangeExistingHoldings,
```

Comment:
If you happen to fancy sorting these in alphabetical order I won't complain :-) but I know it's not in the remit of this work.

Reply:
<img width="183" alt="Screenshot 2021-02-10 at 11 20 32" src="https://user-images.githubusercontent.com/1677737/107503587-0c424900-6b92-11eb-83b0-93c18c977e25.png">

Easily done :)

Reply:
Fixed